docs/client/error_handling.md

```diff
@@ -0,0 +1,62 @@
+# <img align="center" src="../images/logo.png">  Error Handling
+
+## General
+
+Our generated clients raise exceptions defined in [`azure-core`][azure_core_exceptions]. While the base for all exceptions is [`AzureError`][azure_error],
+[`HttpResponseError`][http_response_error] is also a common base catch-all for exceptions, as these errors are thrown in the case of a request being made, and a non-successful
+status code being received from the service.
+
+Our generated code also offers some default mapping of status codes to exceptions. These are `401` to [`ClientAuthenticationError`][client_authentication_error], `404` to
+[`ResourceNotFoundError`][resource_not_found_error], and `409` to [`ResourceExistsError`][resource_exists_error].
+
+A very basic form of error handling looks like this
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.pets import PetsClient
+
+client = PetsClient(credential=DefaultAzureCredential())
+try:
+    dog = client.get_dog()
+except HttpResponseError as e:
+    print("{}: {}".format(e.status_code, e.message))
+```
+
+## Logging
+
+Our generated libraries use the standard [`logging`][logging] library for logging. Basic information about HTTP sessions (URLs, headers, etc.) is logged at INFO level.
+Our logger's name is `azure`.
+
+Detailed DEBUG level logging, including request/response bodies and un-redacted headers, can be enabled on a client with the logging_enable argument:
```

Comment:
wondering if we should include how to create a named logger for your specific lib